src/Nethermind/Nethermind.Core/Bloom.cs

```diff
@@ -138,6 +140,11 @@ public void Add(LogEntry[] logEntries, Bloom blockBloom = null)
                 }
             }
         }
+
+        public void Accrue(Bloom bloom)
```

Comment:
why not Or?

Reply:
note that internally Or on span also works on the original span

Reply:
Wanted more domain name, we could also use Accumulate